docs/fixture-format.md

```diff
@@ -105,13 +107,77 @@ See the [Futurelight PRO Slim PAR-7 HCL fixture](../fixtures/futurelight/pro-sli
 
 ### Matrices
 
-Some fixtures have multiple heads: A horizontal bar of LEDs, a pixel lamp with a grid of lights, a fixture consisting of inner and outer rings of LEDs that can be controlled separately, etc. See the [Eurolite LED KLS 801](../fixtures/eurolite/led-kls-801.json) and the "Matrix" category for example fixtures.
+Some fixtures have multiple light beams: A horizontal bar of LEDs, a pixel head with a grid of lamps, a fixture consisting of inner and outer rings of LEDs that can be controlled separately, etc. See the [Eurolite LED KLS 801](../fixtures/eurolite/led-kls-801.json) and the "Matrix" category for example fixtures.
+
+#### Matrix structure
+
+The information how these pixels are arranged is stored in the fixture's `matrix` object: Either by using the x × y × z syntax from `pixelCount` (e.g. [5, 5, 1] for a 5×5 matrix) or by naming each individual pixel in `pixelKeys`, e.g.:
+
+```js
+"pixelKeys": [
+  [
+    [ null,  "Top",     null  ],
+    ["Left", "Center", "Right"],
+    [ null,  "Bottom",  null  ]
+  ]
+]
+```
+
+`null` refers to a "hole", i. e. there's no light beam, which allows for non-cubic frames. The above example represents 5 heads arranged like a "+".
+
+Pixels can also be grouped if a fixture allows control in different fine grades, like fourths or halfs of a light bar:
```

Comment:
Please also add a note about pixel groups for rings (like the [Robe Robin 1200](http://rdm.openlighting.org/model/display?manufacturer=21075&model=84))